File: python/vectorizematrix.py
#!/usr/bin/env python
# File: vectorizematrix.py
# Name: D.Saravanan
# Date: 27/11/2022

""" Vectorize matrix calculations """

import numpy as np

n = 256
a = np.random.random((n, n))
b = np.random.random((n, n))
c = np.zeros((n, n))

c = c + a * b
print(c)

from numpy.random import rand

m = 256
x = rand(n, n)
y = rand(n, n)
z = np.zeros((n, n))

z = z + x * y
print(z)

File: julia/vectorizematrix.jl
#!/usr/bin/env julia
# File: vectorizematrix.jl
# Name: D.Saravanan
# Date: 20/10/2022

""" Vectorize matrix operations """

using Random

n = 256
a = rand(n, n)
b = rand(n, n)
c = zeros(n, n)

c = c + a * b
println(c)
